packages/design-tokens/src/scss/themes/theme-light-default/_colors.scss

```diff
@@ -151,6 +151,12 @@ $form-field-warning: #c26400 !default;
 $gradient-basic-overlay-color-01: #fff !default;
 $gradient-basic-overlay-color-02: #fff0 !default;
 
+$gradient-primary-color-01: #7847ff !default;
```

Comment:
The colours here are not ideal. I would expect the text-primary text to work on this background, but it does not. Maybe we should check this with Jirka?